### ISSUE TITLE
Just use mkimage.sh …

### ISSUE THREAD

**Issue by ypid** [docker-32bit/debian#4]
Hi

I just tried to build a debian jessie base image on a i368 system using the [mkimage.sh](https://github.com/docker/docker/blob/master/contrib/mkimage.sh) script from docker and it just worked … Is [`docker-32bit/debian`](https://github.com/docker-32bit/debian) still needed?

**Comment by dashohoxha** [comment]
I don't know. I have never tried `mkimage.sh`, I don't even understand how it works.
Did you also build and run a container based on this image?
Maybe it is not needed anymore.

**Comment by ypid** [comment]
Yes. I run `apt-get update` and `apt-get install`. I will use this image as base and report back if something does not work on i386. I was using your script initially but since I am now switching all my systems to a locally build base image I also switched the i368 system to mkimage.sh.

mkimage.sh works kind of similar to this repository.

**Comment by mcandre** [comment]
mkimage and other chroot builders are cool. Yes, docker-32bit is still useful, for people who want 32-bit bases and don't know how to build them.

**Comment by jcoffland** [comment]
I found it useful.  More straight forward than `mkimage.sh` for the particular task of creating a 32-bit Debian image, IMO.

**Comment by ypid** [comment]
I _simpified_ the the usage of `mkimage.sh` using GNU Make: https://github.com/ypid/docker-makefile

Just call `make build-debian-base-image`

Maybe you guys find this useful.

**Comment by dashohoxha** [comment]
@ypid 
Honestly, using a tool to wrap the usage of another tool, in my opinion adds to complexity.
You have a tool X but you have a problem with it. To handle it you use another tool Y, and now you have two problems.
At least it does not help to understand what is going on.

**Comment by ypid** [comment]
There is no problem with this tool, but I understand what you mean. I just wanted to mention it. You are right. Your script is dead simple and is probably useful for some people.
